python/runtime/cudaq/platform/py_alt_launch_kernel.cpp

```diff
@@ -308,17 +317,38 @@ py::object pyAltLaunchKernelR(const std::string &name, MlirModule module,
         std::free(rawArgs);
         return py::int_(concrete);
       })
+      .Case([&](ComplexType ty) -> py::object {
+        auto eleTy = ty.getElementType();
+        return llvm::TypeSwitch<mlir::Type, py::object>(eleTy)
+            .Case([&](Float64Type eTy) -> py::object {
+              std::complex<double> concrete;
+              std::memcpy(&concrete, ((char *)rawArgs) + returnOffset, 16);
```

Comment:
You could share some code here. The FloatXXType classes have a `getIntOrFloatBitWidth()` member function. There is also a `cudaq::opt::convertBitsToBytes()` function. So,
```c++
cudaq::opt::convertBitsToBytes(someTy.getIntOrFloatBitWidth())`
```
would be fully generic here.